src/webgpu/api/operation/storage_texture/read_only.spec.ts

```diff
@@ -18,6 +18,7 @@ import {
   kTextureFormatInfo,
 } from '../../../format_info.js';
 import { GPUTest } from '../../../gpu_test.js';
+import { ShaderStage } from '../../../shader/validation/decl/util.js';
```

Comment:
nit: it is surprising that we introduce a dependency between api/operation and shader/validation. Could we have this utility moved in some common place instead (could be a follow-up).